CONTRIBUTING.md

```diff
@@ -8,8 +8,8 @@ After forking to your own github org or account, do the following steps to get s
 
 ```bash
 # prerequisites
-node --version <= 16.13.1   (for version management, get nvm [recommended])
-java sdk --version <= 11    (for version management, get jenv [optional])
+node --version = 18.17.1   (for version management, get nvm [recommended])
+java sdk --version = 11    (for version management, get jenv [optional])
```

Comment:
It still works on Node 16 and higher JDK versions. Are they a prerequisite?

Reply:
Nyagami's plugin branch works on node 18+ at this point, so I just picked a random LTS number that works with both versions. 

So should I change it back to 16 or...?